src/app/components/BibPage/BibDetails.jsx

```diff
@@ -148,7 +148,9 @@ const BibDetails = (props) => {
           }
           dir={stringDirection(linkText, useParallels)}
         >
+          <Link>
```

Comment:
In order for this to work, you may need to remove the `href` from the `DSLink` and add it to React Router's `Link` with the `to` prop, `<Link to={bibValue.url}>`.

Reply:
i removed the Link entirely